src/streaming/MediaPlayer.js

```diff
@@ -1920,6 +1920,33 @@ function MediaPlayer() {
     }
 
     /**
+     * Get the value of useDeadTimeLatency in AbrController. @see setUseDeadTimeLatencyForAbr
+     *
+     * @returns {boolean=}
+     *
+     * @memberof module:MediaPlayer
+     * @instance
+     */
+    function getUseDeadTimeLatencyForAbr() {
+        return abrController.getUseDeadTimeLatency();
+    }
+
+    /**
+     * Set the value of useDeadTimeLatency in AbrController. If true, only the download
+     * portion will be considered part of the download bitrate and latency will be
+     * regarded as static. If false, the reciprocal of the whole transfer time will be used.
+     * Defaults to true.
+     *
+     * @param {boolean=} useDeadTimeLatency - True or false flag.
+     *
+     * @memberof module:MediaPlayer
+     * @instance
+     */
+    function setUseDeadTimeLatencyForAbr(useDeadTimeLatency) {
+        abrController.setUseDeadTimeLatency(useDeadTimeLatency);
+    }
```

Comment:
And yeah, moving everything to a `config` object accessible on the MediaPlayer's API and whose properties can be accessed and changed directly would be a good idea too ! I agree about the getter/setter mess.
This said, this is a breaking change in the API, so it should be rolled out with care. Maybe in a v3 with API overhaul if other needs for API changes have been expressed.